### DISCUSSION TITLE
Accessing probabilities from a model as a trade condition

### DISCUSSION THREAD

**Discussion by ghost** [edtechre/pybroker#151] [Q&A] [answered]
Hi,

Trying to access the proba from a randomforest classifier in my back test to use as part of the condition to enter, e.g only enter if above 0.6 but having no luck - anyone tried similar with success?

Short of training model separately and registering as a new column which defeats purpose of having all in one place, I can't think of any other any ideas?

**Comment by edtechre** [comment, marked as answer]
Hi @markxbaker,

Have you tried passing a custom predict_fn to model()?  https://www.pybroker.com/en/latest/reference/pybroker.model.html#pybroker.model.model

**Reply by ghost** [reply]
Thanks Ed:)
